src/keyboard.js

```diff
@@ -1,13 +1,13 @@
-const ipc = require('electron').ipcRenderer;
+import { ipcRenderer } from 'electron';
```

Comment:
Don't need to change this, but just so you know, you could've used `import { ipcRenderer as ipc } from 'electron'` to keep the name!